sftp-client.c

```diff
@@ -1404,6 +1468,29 @@ do_download(struct sftp_conn *conn, const char *remote_path,
 			status = SSH2_FX_FAILURE;
 		else
 			status = SSH2_FX_OK;
+
+#ifdef IRONCORE
+		/*  After downloading a file, determine whether it is encrypted or not. Because of the gyrations above,
+		 *  it should have a ".iron" suffix. If the file can be decrypted, the output file will have the ".iron"
+		 *  stripped. If the file wasn't encrypted, we should try to rename without the .iron suffix.
+		 */
+		char dec_fname[PATH_MAX + 1];
```

Comment:
It seems like `PATH_MAX` is supposed to include the null.

See the comment on https://github.com/torvalds/linux/blob/master/include/uapi/linux/limits.h#L12